action.yml

```diff
@@ -0,0 +1,106 @@
+name: Terraform Cloud Workspace
+description: Manages Terraform Cloud workspaces
+inputs:
+  terraform-version:
+    description: Terraform version
+    default: "1.0.3"
```

Comment:
Would be interesting to support semver ranges here. If users choose to run the action on every push to the main branch this would help automate upgrades. And if you want to programmatically roll out new TF versions in bulk to repos, ranges prevent the config from always reverting that.

Reply:
i was hoping a version constraint would be supported out of the box with the [tfinstall](https://github.com/hashicorp/terraform-exec/tree/main/tfinstall) portion of terraform-exec but it looks like could resolve a version separately and pass it in to tfinstall.ExactVersion